src/parser/creative_linear_parser.js

```diff
@@ -303,6 +306,25 @@ function parseIcon(iconElement) {
           url: parserUtils.parseNodeText(iconClickTrackingElement),
         });
       });
+
+    const iconClickFallbackImagesElement = parserUtils.childByName(
+      iconClicksElement,
+      'IconClickFallbackImages'
+    );
+
+    if (iconClickFallbackImagesElement) {
+      parserUtils
+        .childrenByName(
+          iconClickFallbackImagesElement,
+          'IconClickFallbackImage'
+        )
+        .forEach((iconClickFallbackImageElement, id) => {
+          icon.iconClickFallbackImages.push({
+            id,
```

Comment:
Why adding the `id` attribute ? As there is no reference to it in the VAST specs